ClosedXML/Excel/Sparkline/XLSparkLine.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+
+namespace ClosedXML.Excel
+{
+    internal class XLSparkline : IXLSparkline
+    {        
+        public IXLSparklineGroup SparklineGroup { get; }
+
+        public XLFormula Formula { get; set; }
```

Comment:
I think we should store the reference to the data source range (`IXLRange DataSourceRange`) too. This will make the shifting easy. The only usecase when this is not enough is when the source is defined as the named range. @igitur maybe `IXLNamedRange` must inherit `IXLRange` too?